<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginManagerDialog</class>
 <widget class="QDialog" name="PluginManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>405</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plugin Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox::indicator:checked,QGroupBox::indicator:checked{    
	image: url(:/ButtonImage/src/checkbox_hookicon_normal_16px.png);
    border: 0px;
}

QCheckBox::indicator:checked:disabled,QGroupBox::indicator:checked:disabled{
    image: url(:/ButtonImage/src/checkbox_hookicon_disabled_16px.png);
    border: 0px;
}

QCheckBox::indicator,QGroupBox::indicator{
    image: url(:/ButtonImage/src/checkbox_hookicon_unchecked_16px.png);
    border: 0px;
}

QCheckBox::indicator,QGroupBox::indicator{
    image: url(:/ButtonImage/src/checkbox_hookicon_unchecked_16px.png);
    border: 0px;
}

QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 2px 1px 2px;
	background-color: rgb(255, 255, 255);
    width: 60px;
	font: 9pt &quot;方正黑体简体&quot;;
    text-align: center;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
 	background-color: rgb(255, 255, 255);
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:hover {
    background-color: rgb(227, 227, 227);
    /*border: 1px solid rgb(227, 227, 227);*/
}
 
QComboBox::down-arrow {    
    image: url(:/ButtonImage/src/downarrow_16px.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(100, 100, 100);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>24</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>方正黑体简体</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(240, 240, 240);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plugin Manager&lt;/p&gt;&lt;p&gt;———————&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>方正黑体简体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(240, 240, 240);</string>
          </property>
          <property name="text">
           <string>Sort by :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(252, 252, 252);</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="pluginListWidget">
     <property name="styleSheet">
      <string notr="true">QTableView::item{
	color: rgb(64, 64, 64);
	background-color: rgb(220, 220, 220);
	alternate-background-color: rgb(200, 200, 200);
	gridline-color: rgb(120, 120, 120);
}

QHeaderView::section{
	background-color: rgb(90, 90, 90);
	color: rgb(245, 245, 245);
    border: 0px;
    min-height: 30px;
}

QTableView::item:selected{
    background-color: #e3f2fd;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>128</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
    background-color: rgb(230, 230, 230);
}

QPushButton{
    background: rgb(80, 80, 80);
    color: rgb(255, 255, 255);
    border-radius: 12px;
    padding: 3px 14px 3px 14px;
    height: 24px;
	font: 10pt &quot;方正黑体简体&quot;;
}

QPushButton:hover{
    background: rgb(100, 100, 100);
}

QPushButton:disabled{
    background: rgb(170, 170, 170);
}

QPushButton#pushButton_PluginSettingPanel_Uninstall{
    background: #f44336;
    color: rgb(255, 255, 255);
    border-radius: 12px;
    padding: 3px 14px 3px 14px;
    height: 24px;
	font: 10pt &quot;方正黑体简体&quot;;
}

QPushButton#pushButton_PluginSettingPanel_Uninstall:hover{
    background: #ef5350;
}

QPushButton#pushButton_PluginSettingPanel_Uninstall:disabled{
    background: #e57373;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="pluginInfoPanel" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>140</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{
    background-color: rgb(218, 218, 218);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../Resources.qrc">:/Others/src/Plugin.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Name :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_PluginSettingPanel_Name">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string notr="true">( Name )</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Author :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_PluginSettingPanel_Author">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string notr="true">( Author )</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Build Date :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_PluginSettingPanel_BuildDate">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string notr="true">( Build Date )</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_15">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Version :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_PluginSettingPanel_Version">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string notr="true">( Version )</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Description :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_PluginSettingPanel_Description">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string notr="true">( Description )</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>UUID :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_PluginSettingPanel_UUID">
               <property name="font">
                <font>
                 <family>方正黑体简体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string notr="true">( UUID )</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_PluginSettingPanel_HideOrShow">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>10</pointsize>
               <weight>7</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 63 10pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>▼ Hide</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_PluginSettingPanel_Option">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Option</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_PluginSettingPanel_Enable">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Enable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_PluginSettingPanel_Uninstall">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>Uninstall</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
